python/ray/dashboard/client/src/pages/dashboard/NodeRowGroup.tsx

```diff
@@ -60,7 +61,7 @@ class NodeRowGroup extends React.Component<
   State
 > {
   state: State = {
-    expanded: false
+    expanded: (this.props.numNodes > 5 ? false : true) 
```

Comment:
As per discussion with @pcmoritz, let's change the threshold to 4 nodes instead of 5. Also, `this.props.numNodes <= 4` would already return a boolean expression, so we don't need to use a ternary expression here.